server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/WebSocketRoutes.scala

```diff
@@ -115,11 +118,8 @@ class WebSocketRoutes @Inject() (@Named(InjectionKeys.RFC_8621) val authenticato
     val context = ClientContext(sink, new AtomicReference[Registration](), session)
     val responseFlux: SFlux[OutboundMessage] = SFlux[WebSocketFrame](in.aggregateFrames()
       .receiveFrames())
-      .map(frame => {
-        val bytes = new Array[Byte](frame.content().readableBytes)
-        frame.content().readBytes(bytes)
-        new String(bytes, StandardCharsets.UTF_8)
-      })
+      .filter(frame => frame.isInstanceOf[TextWebSocketFrame])
```

Comment:
It will be better if we have a clock watch timer here. 
Eg: if server does not receive `TextWebSocketFrame` or `PongWebSocketFrame` for each "interval duration", 
The server can close the connection and clear the context.